### PR TITLE
Linked Courses related fixes

### PR DESCRIPTION
**Pull request by tuomas777**

### CODE REVIEW

File: src/validation/validationRules.js [hunk 1]
```diff
@@ -131,7 +131,7 @@ var validations = {
     isDate: function isDate(values, value) {
         return validations.matchRegexp(values, value, /^[0-9]{4}\-[0-9]{2}\-[0-9]{2}$/i);
     },
-    afterStartTime: function afterStartTime(values, value) {
+    afterStartTime: function afterStartTime(values, value, key) {
```
Comment (Rikuoja) [inline src/validation/validationRules.js]: What's this unused key parameter?

File: src/validation/validationRules.js [hunk 2]
```diff
@@ -143,6 +143,10 @@ var validations = {
 
         return false;
     },
+    afterEnrolmentStartTime: function afterStartTime(values, value, key) {
```
Comment (Rikuoja) [inline src/validation/validationRules.js]: Same here.

File: src/validation/validationRules.js (hunk 1 above)
Comment (tuomas777) [reply src/validation/validationRules.js]: Good question. Fixed.

File: src/validation/validationRules.js (hunk 2 above)
Comment (tuomas777) [reply src/validation/validationRules.js]: Fixed.

**Comment by tuomas777** [comment]
Added two new commits.

**Pull request merged**